lib/cli/src/commands/app/deploy.rs

```diff
@@ -226,7 +230,11 @@ impl AsyncCliCommand for CmdAppDeploy {
         let client =
             login_user(&self.api, &self.env, !self.non_interactive, "deploy an app").await?;
 
-        let base_dir_path = self.dir.clone().unwrap_or(std::env::current_dir()?);
+        let base_dir_path = self
+            .dir
+            .clone()
+            .unwrap_or(self.path.clone().unwrap_or(std::env::current_dir()?));
```

Comment:
Nit: these need to be `.unwrap_or_else(|| ...)`.

This should fail clippy lints, I'm quite sure.

Reply:
[b0715e4](https://github.com/wasmerio/wasmer/pull/4861/commits/b0715e4833eb07582554f99c375fff29c7b73f46)